pkg/result/filter.go

```diff
@@ -168,6 +155,7 @@ func filterMisconfigurations(result *types.Result, severities []string, includeN
 			continue
 		} else if ignoreMisconfs.Match(result.Target, misconf.ID) || ignoreMisconfs.Match(result.Target, misconf.AVDID) {
 			// Filter misconfigurations by ignore file
+			ignored++
```

Comment:
I might be missing something. Why don't we increment `summary.Exceptions` here?

Reply:
You're right, I missed it. We can simplify this further.